devtools/steam_achievements.py

```diff
@@ -31,31 +31,56 @@ def cleanup_text(text):
 try:
 	session = HTMLSession()
 	response = session.get(statsurl)
-	achievements_rows = response.html.xpath("//tr[starts-with(@id, 'achievement-')]/td")
 	game = response.html.xpath("//h1[@itemprop='name']/text()")
+
+	achievements_rows = response.html.xpath("//tr[starts-with(@id, 'achievement-')]/td")
 	achievements_columns = 3 # id, text, img
-	entries = int(len(achievements_rows) / achievements_columns)
-	if entries == 0:
+	achievements_entries = int(len(achievements_rows) / achievements_columns)
+	if achievements_entries == 0:
 		sys.exit(127)
 
 	if args.verbose:
-		sys.stderr.write('found {0} achievements\n'.format(entries))
+		sys.stderr.write('found {0} achievements\n'.format(achievements_entries))
+
+	stats_rows = response.html.xpath("//tr[starts-with(@id, 'stat-')]/td")
+	stats_columns = 3 # id, text, default value, your value
+	stats_entries = int(len(stats_rows) / stats_columns)
+
+	if args.verbose:
+		sys.stderr.write('found {0} stats\n'.format(stats_entries))
 
 	scummvm_game_id = args.gameid
 	if not scummvm_game_id:
 		scummvm_game_id = game[0].lower().replace(' ', '').replace('-', '')
 		if args.verbose:
 			sys.stderr.write('missing scummvm game id - assuming {0}\n'.format(scummvm_game_id))
 
-	print("\t{\n\t\t\"%s\",\n\t\tCommon::STEAM_ACHIEVEMENTS,\n\t\t\"%s\",\n\t\t{" % (scummvm_game_id, args.steamid))
-	for i in range(entries):
+	print("\t{\n\t\t\"%s\",\n\t\tCommon::STEAM_ACHIEVEMENTS,\n\t\t\"%s\"," % (scummvm_game_id, args.steamid))
```

Comment:
It would be sweet if you could update the twine achievement table, too - as it would otherwise produce an invalid table for lba1 enhanced achievements.

Reply:
Good idea. Added at 6affb8e